### ISSUE TITLE
Links on the top nav bar that point to external sites need to be updated

### ISSUE THREAD

**Issue by tristandenyer** [a85/POSTMan-Chrome-Extension#8136]
**Describe the bug**
The links in the top nav menubar and About dialog modal for Mac version of app are incorrect and cause a redirect. This redirect wipes clean all referring data in Google Analytics. 

**To Reproduce**
1. Open Mac Postman app
2. Click on 'Help' in top nav bar
3. Click on any of the provided links
4. notice that the browser pages quickly redirect to resolved page

**Expected behavior**
links should point to:
Documentation: https://learning.postman.com/
GitHub: https://github.com/postmanlabs/postman-app-support/
Twitter: https://twitter.com/getpostman
Support: https://www.postman.com/support

In About dialog modal:
Our website: https://www.postman.com/
Twitter: https://twitter.com/getpostman
Documentation: https://learning.postman.com/

**Screenshots**
<img width="444" alt="Screen Shot 2020-03-03 at 1 01 31 PM" src="https://user-images.githubusercontent.com/4358288/75819877-3b302800-5d50-11ea-86b1-645fb92637f7.png">

<img width="336" alt="Postman-app-about-dialog" src="https://user-images.githubusercontent.com/4358288/75820289-ec36c280-5d50-11ea-8442-c3da3d2ff223.png">

**App information (please complete the following information):**
 - Native app
 - Postman Version Version 7.19.1 (7.19.1)
 - OS: Catalina 10.15.3 (19D76)

**Additional context**
Basically, the links on the top nav bar that point to external sites all need to be updated to reflect new domain changes and give us referrer data.

**Comment by kamalaknn** [comment]
All links are modified now so they have a single redirect.
